.github/workflows/ci.yaml

```diff
@@ -20,6 +20,7 @@ jobs:
         os: [macOS-latest, ubuntu-latest]
         python-version: [3.7]
     env:
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: true
```

Comment:
Just curious why this is needed now.

Reply:
It was giving me this error and it's related to an update to Github Actions. Adding this to the `ci.yaml` file fixed it.
![image](https://user-images.githubusercontent.com/47799290/99607770-536cd380-29c1-11eb-8167-18f96469ee69.png)